python/python/glide/async_commands/core.py

```diff
@@ -1601,6 +1601,29 @@ async def sunionstore(
             await self._execute_command(RequestType.SUnionStore, [destination] + keys),
         )
 
+    async def sinter(self, keys: List[str]) -> Set[str]:
+        """
+        Gets the intersection of all the given sets.
+        When in cluster mode, all keys must map to the same hash slot.
```

Comment:
Can you add a blank line between this line and the one above, also we recently changed the format of this notice, can you change to align with the new format (should have "Note:" above)? You can see an example [here](https://github.com/aws/glide-for-redis/blob/main/python/python/glide/async_commands/core.py#L3247)